services/records/record_templates/schemas.go

```diff
@@ -0,0 +1,167 @@
+package record_templates
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/leanspace/terraform-provider-leanspace/helper/general_objects"
+)
+
+var validRecordTemplatesConstraintTypes = []string{"UNKNOWN", "PASSED", "FAILED"}
```

Comment:
To be used for the state field of the schema ?

Reply:
This properties should be used for the validation of the State. Since the state is only generated and can't be modified for now, they was not used. So I've remove them.